src/components/Page/components/Header/components/Title/Title.tsx

```diff
@@ -59,17 +47,10 @@ export function Title({
     </div>
   ) : null;
 
-  const thumbnailMarkup = thumbnail ? <div>{thumbnail}</div> : null;
-
-  const pageTitleClassName = thumbnail ? styles.hasThumbnail : undefined;
-
   return (
-    <div className={pageTitleClassName}>
-      {thumbnailMarkup}
-      <div className={styles.TitleAndSubtitleWrapper}>
-        {wrappedTitleMarkup}
-        {subtitleMarkup}
-      </div>
+    <div>
```

Comment:
Can we use a fragment here?
```suggestion
    <>
```

Reply:
I want to investigate before merging. My gut feeling is no because there is some layout that wraps the title.